test/test_suit/resilience_tests/test_kc_delete_create_pressure.py

```diff
@@ -0,0 +1,76 @@
+import datetime
+
+from test_suit.test_utils import RetryableError, NonRetryableError, ResetAndRetry
+import json
+from time import sleep
+
+class TestKcDeleteCreatePressure:
```

Comment:
whats the difference between this and `TestKcDeleteCreate`? 
Pressure test as far as I know does a copy unload from a table and reloads it into topic. I see only 1k records. I was wondering if we could combine those two? (Just to reduce the e2e test and noise if there is something wrong)

Reply:
"pressure" is just a test that sends data between the two methods. I made the distinction because the customer's issue with v1.9.0 only happens if there is data ingesting while the connector is paused then resumed. Just pausing/resuming the connector does not raise an error

test_kc_delete_create_pressure: delete connector, send data, create connector
test_kc_delete_create: delete connector, create connector

Reply:
Got it, thanks for explaning! 
Lets add a description on each file in python class.